src/autoscaler/api/default_info.json

```diff
@@ -0,0 +1,8 @@
+
+
+{
+  "name": "Autoscaler",
+  "build": "14.3.0",
```

Comment:
how do we update this build no ? is there any automation around it

Reply:
answer is here https://github.com/cloudfoundry/app-autoscaler-release/pull/3119/files#diff-4f334133a86ee5ab13bd038dca13682d12567bfc17674612c5a7ee8cb07f91e1R36